packages/dds/tree/src/simple-tree/core/withType.ts

```diff
@@ -14,19 +16,44 @@
  * Instead construct a real node of the desired type using its constructor.
  * @privateRemarks
  * This prevents non-nodes from being accidentally used as nodes, as well as allows the type checker to distinguish different node types.
- * @public
+ * @deprecated External code should use `Tree.schema(theNode)` for runtime data access, and for typechecking and internally {@link typeSchemaSymbol} provides a superset of this functionality.
```

Comment:
I'm having a hard time parsing this after "for runtime data access,".

Reply:
Rephrased.

Reply:
Looks good! Thanks!